internal/oci/runtime_oci.go

```diff
@@ -1340,6 +1340,8 @@ func prepareProcessExec(c *Container, cmd []string, tty bool) (processFile strin
 	if tty {
 		pspec.Terminal = true
 	}
+	// Allow the same capabilities to be used as those from the actual container process.
+	pspec.Capabilities.Inheritable = pspec.Capabilities.Bounding
```

Comment:
Is there a need to check if there's a value in `pspec.Capabilities.Inheritable` first, or will it always be empty?

Reply:
I think we can check for `pspec.Capabilities != nil` before setting the field to be on the safe side.